NEA.py

```diff
@@ -14,96 +14,105 @@
  ____________________________________                                    
 '''
 
-print banner
+print(banner)
 
 try:
-    import datetime,requests,re
+    import datetime
+    import requests
+    import re
 except ModuleNotFoundError:
-	print '\n\033[91m[-] Module not found. Try pip install -r requirements.txt'
-	exit()
+    print('\n\033[91m[-] Module not found. Try pip install -r requirements.txt')
+    exit()
 
 
 x = datetime.datetime.now()
 
 d = x.strftime("%b-%y").upper()
 # print current date
 date = x.strftime("%m/%d/%Y")
-print "\033[97m           | " + date + ' |'
+print("\033[97m           | " + date + ' |')
 
 
-sc = raw_input("\n\033[92m Enter your Sc. No. : \033[94m")
-custid = raw_input("\033[92m Enter Customer id: \033[94m")
+sc = input("\n\033[92m Enter your Sc. No. : \033[94m")
+custid = input("\033[92m Enter Customer id: \033[94m")
 
-add = raw_input("\033[92m Enter your NEA Location : \033[94m").upper()
+add = input("\033[92m Enter your NEA Location : \033[94m").upper()
 
 file = 'address.txt'
 
+
 def exit2():
 
-	print('\n\033[94m [+] Our Site: www.askbuddie.com\n [+] Join Us: www.fb.com/groups/askbuddie\n [+] Our FB page: www.fbcom/askbuddie\n [+] ABOS: https://github.com/askbuddie')
-	print('\n\033[93m Thanks for using NEA Bill Checker...\n\n')
-	exit()
+    print('\n\033[94m [+] Our Site: www.askbuddie.com\n [+] Join Us: www.fb.com/groups/askbuddie\n [+] Our FB page: www.fbcom/askbuddie\n [+] ABOS: https://github.com/askbuddie')
+    print('\n\033[93m Thanks for using NEA Bill Checker...\n\n')
+    exit()
+
 
```

Comment:
Error handling while reading address.txt.
```suggestion
try:
    a = open(file)
except Exception:
    print(f"\n\033[91m [-] {file} not found.")
    exit2()
```